src/BlackHoleOptions.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cache for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cache/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cache/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Laminas\Cache\Storage\Adapter;
+
+final class BlackHoleOptions extends AdapterOptions
```

Comment:
The check for the PSR compatibility is only needed in this storage adapter?

Reply:
Yes, thats a workaround to avoid BC break.
Most other adapters are already PSR compatible and tested with integration tests.